infra/src/main/kotlin/fi/fta/geoviite/infra/geometry/GeometryDao.kt

```diff
@@ -1349,58 +1343,6 @@ class GeometryDao @Autowired constructor(
         }
     }
 
-    @Transactional
-    fun removeReferencesToTrackNumber(id: IntId<TrackLayoutTrackNumber>): RemovedTrackNumberReferenceIds {
```

Comment:
Old reference clearing logic removed and replaced with references built as "on delete set null".